lhotse/cut.py

```diff
@@ -2135,6 +2208,51 @@ def perturb_volume(self, factor: float, affix_id: bool = True) -> "MixedCut":
             ],
         )
 
+    def reverb_rir(
+        self,
+        rir_cut: "MonoCut",
```

Comment:
Is there a reason to pass RIR as a cut instead of as a recording? Do you expect to truncate / pad them somehow?

Reply:
Yeah, good point. I passed it as a Cut so it may be possible to support transformations on the RIR in the future. But you're right, perhaps it's best to use it as a Recording for now.